docs/elasticsearch.md

```diff
@@ -48,3 +48,17 @@ Copy the mapped IP and port (highlighted in bold above) and use it to query Elas
 ```
 curl -XGET http://0.0.0.0:32871
 ```
+
+## Configuration updates when using ElasticSearch on Local Environment
```

Comment:
Sorry I must've made an error on the issue, the issue isn't to do with the local environment but rather continuous integration environments like Travis and CircleCI and Ubuntu operating systems.

The purpose of this section is just to document the environment variable so can you make the title:
```suggestion
## ElasticSearch Memory Limit
```

Reply:
This was my error - I had in my notes it was for the continuous integration environments but when I rewrote I used a section just below that, basically cutting it from the notes. Sorry about that.

Reply:
With the suggestions you can commit them from the UI directly here - looks like you marked it resolved before doing that :)